BCnEnc.Net/Shared/DdsFile.cs

```diff
@@ -460,8 +460,9 @@ public DxgiFormat DxgiFormat
 								{
 									return DxgiFormat.DxgiFormatR8G8B8A8Unorm;
 								}
-								else if (dwRBitMask == 0xff0000 && dwGBitMask == 0xff00 && dwBBitMask == 0xff &&
-										 dwABitMask == 0xff000000)
+
+								if (dwRBitMask == 0xff0000 && dwGBitMask == 0xff00 && dwBBitMask == 0xff &&
+								    dwABitMask == 0xff000000)
```

Comment:
Why change the else if here?

Reply:
The else is unnecessary, since a return happens in the if. And functionally both lines are equal. So mainly formatting.